docs/docsite/_themes/srtd/layout.html

```diff
@@ -38,6 +38,9 @@
   <link href='https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700' rel='stylesheet' type='text/css'>
   <link href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.0.3/css/font-awesome.min.css' rel='stylesheet' type='text/css'>
 
+  {# CANONICAL URL #}
+  <link rel="canonical" href="https://docs.ansible.com/ansible/latest/index.html">
+  
```

Comment:
Is this not going to result in **all** 2.3 results referencing the index page for latest, including all module docs?

Will these pages be "near identical" enough that Google et al will actually default to the "latest" versions?

Reply:
found how this was solved in 2.4 https://github.com/ansible/ansible/pull/53801 so using that cherry-pick instead. seems to work.